app/styles/_toast.less

```diff
@@ -0,0 +1,16 @@
+//
+// Toast alerts
+// -------------------------------------------------
+
+.alert.toast-pf {
```

Comment:
I think we'd only want to change overview alerts, not all toast notifications, which we might start using someday.